docs/static/cross-plugin-concepts.asciidoc

```diff
@@ -0,0 +1,17 @@
+[[plugin-concepts]]
+=== Cross-plugin concepts and features
+
+New section for concepts, features, and behaviors that apply to multiple plugins.
```

Comment:
Standardize on either US or EU spelling  of behavior/behaviour (lines 4 and 10). Your choice. :-) Otherwise, build cleanly and LGTM

Reply:
```suggestion
New section for concepts, features, and behaviours that apply to multiple plugins.
```